install/kubernetes/cilium/values.yaml

```diff
@@ -735,6 +735,11 @@ hubble:
     # -- Whether to enable the Hubble UI.
     enabled: false
 
+    # -- When true, it will allow installing the Hubble UI only, without checking dependencies.
+    # It is useful if a cluster already has cilium and Hubble relay installed and you just
+    # want Hubble UI to be deployed.
+    allowStandalone: false
```

Comment:
Please run `make -C install/kubernetes docs` to update `install/kubernetes/cilium/README.md`.

Reply:
Done ✅

Reply:
Thanks for pointing out the other PR, didn't notice it.
I have a couple ideas regarding the certificate, i'll try to update the PR by the end of the day 🤞